sgt-back-end/index.js

```diff
@@ -0,0 +1,150 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select *
+    from "grades"`;
+    db.query(sql)
+    .then((result => {
+      res.status(200).json(result.rows);
+    }))
+    .catch(err => {
+      res.status(500).json({
+        error: 'Something went wrong'
+      })
+    })
+})
+
+app.use(express.json())
+
+app.post('/api/grades', (req, res) => {
+  if(req.body.name === undefined || req.body.course === undefined || req.body.score === undefined || req.body.score < 1 || req.body.score > 100) {
+    res.status(400).json({
+      error: 'Something is wrong with your request parameters'
+    })
+    return;
+  }
```

Comment:
`score` also needs to be an integer:

---

![image](https://user-images.githubusercontent.com/16143771/119539905-9e392080-bd41-11eb-804e-577719515f1a.png)